bscript/interpreter/opcode.go

```diff
@@ -0,0 +1,329 @@
+// Copyright (c) 2013-2017 The btcsuite developers
+// Use of this source code is governed by an ISC
+// license that can be found in the LICENSE file.
+
+package interpreter
```

Comment:
Since this ties a parsed op to its respective function, I couldn't move them into bscript without introducing a circular dependency between bscript and the interpreter.

I'll see moving these opcode->operation mapping into the operations.go file